File: Biblioteca/src/main/java/com/biblioteca/domain/obra/TipoObra.java
package com.biblioteca.domain.obra;

public enum TipoObra {
    NOVELA("Novela"),
    ENCICLOPEDIA("Enciclopedia"),
    TRATADO("Tratado"),
    CUENTO_INFANTIL("Cuento infantil");

    
    private String tipo;
    
    private TipoObra(String tipo) {
        this.tipo = tipo;
    }
    public String getTipoObra() {
        return tipo;
    }
}

File: python/biblioteca/src/domain/obra/tipo_obra.py
from enum import Enum
class TipoObra(Enum):
    NOVELA = "Novela"
    COMEDIA = "Comedia"
    ENCICLOPEDIA = "Enciclopedia"
    TRATADO = "Tratado"
    CUENTO_INFANTIL = "Cuento infantil"

    def __str__(self):
        return f"{self.value}"